test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -71,7 +75,11 @@ import {
   BO_GOV_ID_URL,
   MO_IND_ID_URL,
   MO_CORP_ID_URL,
-  TRANSACTION_ID, PUBLIC_REGISTER_NAME, PUBLIC_REGISTER_JURISDICTION, REGISTRATION_NUMBER,
+  TRANSACTION_ID,
+  PUBLIC_REGISTER_NAME,
+  PUBLIC_REGISTER_JURISDICTION,
+  REGISTRATION_NUMBER,
+  BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK, MANAGING_OFFICER_OBJECT_MOCK,
```

Comment:
I'd have a new line after the comma so MANAGING_OFFICER_OBJECT_MOCK is on a new line

Reply:
New line